src/compiler/transformers/ts.ts

```diff
@@ -2034,6 +2064,11 @@ namespace ts {
             );
         }
 
+        function isSimpleComputedPropertyName(expression: Expression) {

```

Comment:
The function name is misleading. Maybe `isSimpleComputedPropertyNameExpression`? Though, that is a bit of a mouthful.